compiler/ast/ast_query.nim

```diff
@@ -106,6 +106,27 @@ const
 
   nkSymChoices* = {nkClosedSymChoice, nkOpenSymChoice}
 
+  nkTypeExprs* = {
+    nkTypeOfExpr,
+    nkObjectTy,
+    nkTupleTy,
+    nkTupleClassTy,
+    nkTypeClassTy,
+    nkStaticTy,
+    nkRefTy,
+    nkPtrTy,
+    nkVarTy,
+    nkConstTy,
+    nkMutableTy,
+    nkDistinctTy,
+    nkProcTy,
+    nkIteratorTy,
+    nkSharedTy,
+    nkEnumTy,
+    nkStmtListType,
+    nkBlockType
```

Comment:
I wonder if we should not include these under expressions?

Reply:
Yeah, it doesn't matter much because they can be removed anyways I think.